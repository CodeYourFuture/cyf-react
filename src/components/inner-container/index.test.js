import React from 'react';
import { shallow } from 'enzyme';

import InnerContainer from './';

describe('InnerContainer Component', () => {
  it('renders correctly', () => {
    expect(shallow(<InnerContainer />)).toMatchSnapshot();
  });
});
