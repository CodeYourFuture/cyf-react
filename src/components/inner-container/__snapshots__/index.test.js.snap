// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InnerContainer Component renders correctly 1`] = `
<div
  className="css-89slzz"
/>
`;
